src/it/resources-noncompilable/org/checkstyle/suppressionxpathfilter/packagedeclaration/SuppressionXpathRegression1.java

```diff
@@ -0,0 +1,5 @@
+package org.junit.contrib; // warn
```

Comment:
each non-compiled file should have comment like
https://github.com/checkstyle/checkstyle/blob/513ebd47dc07dc5b062776ec6cfefa8f5e27575b/src/it/resources-noncompilable/org/checkstyle/suppressionxpathfilter/packageannotation/SuppressionXpathRegressionPackageAnnotationOne.java#L1 to explain a reason.

Please do the same for other Input too.

Reply:
done

Reply:
> non-compilable.

`wrong package`.

Reply:
done.

Reply:
not a comment .
please make code as `package this.package.mismatch.with.folder; // warn`

Reply:
@romani In that case our checkstyle throws error as it thinks this.package. are unknown tokens

Reply:
my.packagename.mismatch.with.folder

Reply:
done